packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -72,29 +69,66 @@ export default {
 		increaseButtonStyle() {
 			return {
 				'kt-input-number__button': true,
-				'kt-input-number__button--disabled':
-					this.currentValue == this.max || this.disabled,
+				'kt-input-number__button--disabled': this.incrementDisabled,
 			}
 		},
 		decreaseButtonStyle() {
 			return {
 				'kt-input-number__button': true,
-				'kt-input-number__button--disabled':
-					this.currentValue == this.min || this.disabled,
+				'kt-input-number__button--disabled': this.decrementDisabled,
 			}
 		},
 	},
 	watch: {
 		value(val) {
 			this.currentValue = parseFloat(val)
```

Comment:
use the setter please

Reply:
if value is changed from without we shouldn't emit, should we?